privilege/privileges/privileges_test.go

```diff
@@ -2058,3 +2058,28 @@ func tearDownTest(t *testing.T, store kv.Storage, dbName string) {
 	dropDBSQL := fmt.Sprintf("drop database if exists %s;", dbName)
 	mustExec(t, se, dropDBSQL)
 }
+
+func TestGrantReferences(t *testing.T) {
+	store, clean := newStore(t)
```

Comment:
Could add `t.Parallel()` at the begining.

Reply:
Thanks for the suggestion, I had some issues during development so I did not add it, but now when the parser is merged I added it.